packages/console/src/components/Pagination/index.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '@/scss/underscore' as _;
+
+.pagination {
+  display: flex;
+  justify-content: right;
+
+  li {
+    list-style: none;
+
+    &:not(:first-child) {
+      margin-left: _.unit(2);
+    }
+
+    .button {
+      border-radius: _.unit(1.5);
+      padding: 0;
+      width: 32px;
+      height: 32px;
```

Comment:
nit: `_.unit(8)`?

Reply:
We only use `_.unit` for padding / margin.